NAMESPACE

```diff
@@ -125,17 +125,21 @@ import(ggplot2)
 import(ggpubr, except = "get_legend")
 import(ggrepel)
 import(ggsci)
-import(ggthemes, except = "theme_map")
 import(gridExtra, except = "combine")
 import(maftools)
 import(metaviz)
 import(plotly, except = c("last_plot", "add_heatmap", "export"))
 import(purrr)
 import(readr)
 import(rtracklayer)
+import(ssh)
+import(stringi)
 import(stringr)
 import(tibble)
 import(tidyr)
 import(vroom, except = c("col_skip", "fwf_positions", "default_locale", "date_names_lang", "cols_only", "output_column", "col_character", "col_guess", "spec", "as.col_spec", "fwf_cols", "cols", "col_date", "col_datetime", "locale", "col_time", "cols_condense", "col_logical", "col_number", "col_integer", "col_factor", "fwf_widths", "date_names_langs", "problems", "date_names", "col_double", "fwf_empty"))
 import(wordcloud)
 import(workflowr)
+importFrom(ggthemes,theme_foundation)
+importFrom(plyr,round_any)
+importFrom(utils,getFromNamespace)
```

Comment:
Since utils are one of those packages that get loaded automatically when opening R studio, does this mean that only the `getFromNamespace` function will be loaded? I am assuming you added this to avoid conflicting function names?

Reply:
Ah I see, thanks for looking into this.